components/CreateCollectionModal.tsx

```diff
@@ -99,10 +102,19 @@ export const CreateCollectionModal = ({
         ) : (
           <Center>Save at least one resource to your collection.</Center>
         )}
+        {!authUser && (
+          <Text>
+            You must{' '}
+            <Text as="span" variant="clickable" onClick={onAuthModalOpen}>
+              create an account or login
+            </Text>{' '}
+            before creating an collection.
+          </Text>
+        )}
```

Comment:
Idk if this is the best UI/UX but it's the easiest to implement - if we get feedback we can come back and update unless anyone has immediate thoughts

Reply:
I agree it would be better to see how Kristen's team reacts to it and we can change it based on that